src/algorithms/graphs/bfs.rs

```diff
@@ -0,0 +1,18 @@
+use std::collections::VecDeque;
+
+#[allow(dead_code)]
+pub fn bfs(adj: &mut Vec<Vec<usize>>, vis: &mut Vec<bool>, v: usize) {
```

Comment:
@Segmentree 
the initial node should be called `s` from the source, don't you think?
adj shouldn't be mutable